images/fcos-bastion-image/root/usr/share/bastion_services/bind9/named.conf

```diff
@@ -0,0 +1,11 @@
+/ This is the primary configuration file for the BIND DNS server named.
+//
+// Please read /usr/share/doc/bind9/README.Debian.gz for information on the
+// structure of BIND configuration files in Debian, *BEFORE* you customize
+// this configuration file.
+//
+// If you are just adding zones, please do that in /etc/bind/named.conf.local
+
+include "/etc/bind/named.conf.options";
```

Comment:
ok, this file should go in the Container image of bind9 itself, no need for this being a volume.

Moreover, the previous image was based on debian, the new one is based on Centos and we do not have the same files in the new one.

You should:
1. Create a root/etc/bind folder in images/bind9
2. Add all the needed files from the bind9 container in the openshift-qe-bastion
3. No need to set /etc/named.conf as a volume
4. Test the bind9 container can run correctly (even locally, not mandatory on fcos)


What we need is the same configuration as in the debian-based image we run currently, or a similar one we are sure we can set the zones as volumes.